src/lib/parse/parseContextAttributeCompoundValue.h

```diff
@@ -0,0 +1,49 @@
+#ifndef SRC_LIB_PARSE_PARSECONTEXTATTRIBUTECOMPOUNDVALUE_H_
```

Comment:
Given that we already have an "style" for V2 directories (started with lib/serviceRoutinesV2), I think we should follow it. It should be only a matter of creating a new directory, move some files and adjust `#includes` and CMakeList.txt, isn't it?